package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "@mongodb-js/zstd",
   "version": "2.0.0-alpha.0",
-  "main": "index.js",
+  "main": "lib/index.js",
   "types": "index.d.ts",
   "repository": "https://github.com/mongodb-js/zstd",
   "files": [
     "index.d.ts",
     "lib/index.js",
-    "addon/*"
+    "addon/*",
+    "README.md",
+    "etc",
+    "HISTORY.md",
```

Comment:
We don't have to ship the changelog, we don't in the driver or bson. (we do in FLE but I think we should take that out)


The rest seems fine (and necessary 😅) though